components/plan-build-ps1/bin/hab-plan-build.ps1

```diff
@@ -287,10 +287,23 @@ function Invoke-Begin {
   Invoke-DefaultBegin
 }
 
-# Default implementation for the `Invoke-Begin` phase.
+# Default implementation for the `Invoke-Before` phase.
```

Comment:
We actually added a [do_before](https://github.com/habitat-sh/habitat/blob/master/components/plan-build/bin/hab-plan-build.sh#L3125) phase which happens after [do_begin](https://github.com/habitat-sh/habitat/blob/master/components/plan-build/bin/hab-plan-build.sh#L3107)

This is a small step to get plan-build.ps1 inline with the most recent plan-build.sh